CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # What's New?
 
+## 1.15
+Features:
+- Added support for the CMake Debugger! [#3093](https://github.com/microsoft/vscode-cmake-tools/issues/3093)
```

Comment:
I can understand the excitement, but we don't usually put exclamations points in the changelog.  It seems more like something for the blogpost.

Reply:
Haha fair point, I'll adjust